extensions/wikia/Wall/WallHistoryController.class.php

```diff
@@ -217,8 +219,18 @@ private function getFormatedHistoryData( $history, $threadId = 0 ) {
 				$history[$key]['msgurl'] = $messagePageUrl;
 
 				$msgUser = $wm->getUser();
+				$msgPage = Title::newFromText( $msgUser->getName(), $ns );
+				if ( empty( $msgPage ) ) {
```

Comment:
@garthwebb did you check why ```WallHostory::get``` returns array that contains objects different than title object?